config/rewrites.d/support.rackspace.com.json

```diff
@@ -62,6 +62,13 @@
         "rewrite": false,
         "status": 301
       },
+      {
+        "description": "Redirect how-to/update-billing-information-for-cloud-hosting/ URL to /retired-articles/update-billing-information-for-cloud-hosting",
+        "from": "^\\/how-to\\/update-billing-information-for-cloud-hosting(\/|)$",
+        "to": "/retired-articles/update-billing-information-for-cloud-hosting",
+        "rewrite": false,
+        "status": 301
+      },      
```

Comment:
generally, our redirects look like the following.       Your code might work anyway...but I recommend being consistent.  Of course, change the description and article name ....
{
        "description": "Retire Administrator management: Cloud Office Control Panel",
        "from": "^\\/how-to\\/administrator-management-cloud-office-control-panel(.*)$",
        "to": "/how-to/article-retired/",
        "rewrite": false,
        "status": 301
      },